.github/workflows/reusable-test.yaml

```diff
@@ -113,3 +113,21 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           path: ./logs
+
+  codespell:
+    runs-on: ubuntu-latest
```

Comment:
Please also add `name: Check spelling` to make this job look pretty in the github actions UI